Documentation/ApiOverview/Mail/Index.rst

```diff
@@ -9,14 +9,24 @@ Mail API
 
 .. versionadded:: 10.2
 
-   Related changelog: :doc:`t3core:Changelog/10.0/Feature-88643-NewMailAPIBasedOnSymfonymailerAndSymfonymime`
+   Symfony mailer and mime support was added in 10.2:
```

Comment:
Do we need to double the information that it was added in 10.2?

Rendering result is right now:
```
New in version 10.2: Symfony mailer and mime support was added in 10.2: Feature: #88643 - New Mail API based on symfony/mailer and symfony/mime
```